packages/types/src/extensions/pn-any-to-erc20-types.ts

```diff
@@ -1,10 +1,12 @@
 import * as PnAnyToAnyConversion from './pn-any-to-any-conversion-types';
+import { EvmChainName } from '../currency-types';
 
 /** Any to ERC20 reference-based payment network extension interface */
 export type IAnyToERC20 = PnAnyToAnyConversion.IConversionReferenceBased<ICreationParameters>;
 
 /** Parameters for the creation action */
 export interface ICreationParameters extends PnAnyToAnyConversion.ICreationParameters {
+  network?: EvmChainName;
```

Comment:
precise `network` type for `pn-any-to-erc20`

Reply:
I'd like to know if @leoslr thinks this is compatible with the approach of calling all fungible tokens `ERC20`, even on non-EVMs.

Reply:
It won't be compatible no. 
Currently the tests are pasing as `aurora` and `aurora-testnet` are declared as `EvmChainName`.

I think we should keep the `ChainName` from the underlying interface. Calling all fungible token ERC20 is a langage abuse, but in the context of creating requests and detecting the payment, the logic remains the same.